atoma-service/src/handlers/mod.rs

```diff
@@ -549,28 +549,29 @@ mod vllm_metrics {
         chat_completions_service_urls: &[String],
         model: &str,
     ) -> Result<(String, StatusCode)> {
+        const MAX_ALLOWED_REQUEST_QUEUE_TIME_SECONDS: f64 = 2.0;
```

Comment:
why is max time 2 when the min request queue time is 4 (change on line 558)
so the min is always larger than max?